frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -98,13 +109,24 @@ class Chart extends Component<Props, State> {
     ) {
       const data = this.state.source === 'hourly' ? chartDataHourly : chartDataDaily;
       this.lineSeries.setData(data);
+      this.setFormattedData(data);
     }
   }
 
   private hasData = (): boolean => {
     return this.props.data.chartDataDaily && this.props.data.chartDataDaily.length > 0;
   };
 
+  private setFormattedData(data: ChartData) {
+    this.formattedData = {} as FormattedData;
+
+    data.forEach( entry => {
+      if (this.formattedData) {
+        this.formattedData[entry.time as number] =  entry.formattedValue;
```

Comment:
if you rebase on upstream/master, a new linter should complain about the double space here when running `make weblint`. if not, we should find a linter that does :D